### ISSUE TITLE
Speed up index creation in modern search

### ISSUE THREAD

**Issue by trishorts** [smith-chem-wisc/MetaMorpheus#412]
get it below 30s

**Comment by rmillikin** [comment]
it would be nice to write the index to the MetaMorpheus folder rather than the data folder, with the database's name and perhaps a revision date as an identifier, so that the user does not have to remake the index every time they want to run a modern search

**Comment by stefanks** [comment]
That is exactly how the index files are being written right now...

**Comment by rmillikin** [comment]
I thought it wrote to a subfolder in the data folder currently?

**Comment by stefanks** [comment]
No, it writes the index file in the same folder as the database, with folder name being the date/time of creation. Slightly off-topic, please create a new issue if this output file location is a problem!
